aireview/src/aireview/application/controllers/review_controller.py

```diff
@@ -16,4 +16,4 @@ def __init__(
     async def review_pull_request(self, pr_id: int) -> None:
         pull_request = await self._github.get_pull_request(pr_id)
         review = self._analyzer.analyze_code(pull_request)
```

Comment:
**style**: Consider adding error handling for get_pull_request call as it could fail due to invalid PR ID or API issues